### PR TITLE
updates unordered writes config docs

### PR DESCRIPTION
**Pull request by owen-d**

### CODE REVIEW

File: docs/sources/configuration/_index.md [hunk 1]
```diff
@@ -1817,6 +1817,10 @@ logs in Loki.
 # CLI flag: -ingester.max-global-streams-per-user
 [max_global_streams_per_user: <int> | default = 0]
 
+# Whether out of order writes are be accepted.
```
Comment (KMiller-Grafana) [inline docs/sources/configuration/_index.md:1820]:
```suggestion
# When true, out of order writes are accepted.
```

File: docs/sources/operations/ordering.md [hunk 2]
```diff
@@ -6,7 +6,7 @@ weight: 60
 
 Enabling out of order writes involves two configuration options in Loki:
 
-- `ingester.unordered-writes-enabled` must be enabled
+- `ingester.unordered-writes` must be enabled (This is also configurable per tenants as part of the [`limits_config`](../configuration#limits_config))
```
Comment (KMiller-Grafana) [inline docs/sources/operations/ordering.md:9]:
```suggestion
- `ingester.unordered-writes` must be enabled. This is also configurable per tenants as part of [`limits_config`](../configuration#limits_config).
```

File: docs/sources/operations/ordering.md [hunk 3]
```diff
@@ -6,7 +6,7 @@ weight: 60
...
 - `ingester.max-chunk-age` (default 1h) is used to parameterize _how far back out of order data is accepted_.
 
-When unordered writes are enabled, Loki will accept older data for each stream as far back as `most_recent_line - (max_chunk_age/2)`. For instance, if `ingester.max-chunk-age=2h` and the stream `{foo="bar"}` has one entry at `8:00`, Loki will accept data for that stream as far back in time as `7:00`. If another log line is wrritten at `10:00`, this boundary changes to `9:00`. Anything farther back than this range will still return an _out of order_ error.
+When unordered writes are enabled, Loki will accept older data for each stream as far back as `most_recent_line - (max_chunk_age/2)`. For instance, if `ingester.max-chunk-age=2h` and the stream `{foo="bar"}` has one entry at `8:00`, Loki will accept data for that stream as far back in time as `7:00`. If another log line is wrritten at `10:00`, this boundary changes to `9:00`. Anything farther back than this range will still return an _out of order** error.
```
Comment (KMiller-Grafana) [inline docs/sources/operations/ordering.md:12]:
```suggestion
When unordered writes are enabled, Loki will accept older data for each stream as far back as `most_recent_line - (max_chunk_age/2)`. For instance, if `ingester.max-chunk-age=2h` and the stream `{foo="bar"}` has one entry at `8:00`, Loki will accept data for that stream as far back in time as `7:00`. If another log line is written at `10:00`,  Loki will accept data for that stream as far back in time as `9:00`. Anything farther back will return an out of order error.
```

**Review by KMiller-Grafana — DISMISSED** [review]

**Review by slim-bean — APPROVED** [review]
LGTM

**Comment by owen-d** [comment]
ref https://github.com/grafana/loki/issues/1544

**Pull request merged**